### PR TITLE
Add getting started section

### LINKED ISSUE #58: Add Getting Started section
Any thoughts on adding a beginners/introduction section?

---------

### Getting started

If this is your first time dealing with transit data, you might find these links useful:

- [GTFS](https://developers.google.com/transit/gtfs/) - GTFS is a of text files that contains infrequently changing transit data, like stops, routes, trips, and other schedule data. Most transit agencies typically update their GTFS feed every few months.
- [GTFS Realtime](https://developers.google.com/transit/gtfs-realtime/) - GTFS Realtime consists of three binary files that contain realtime vehicle positions, realtime arrival information, and service alerts. Transit agencies typically update these files every minute.
- [Transitland ](https://transit.land/) - A community curated list of GTFS/GTFS-realtime data feeds from around the world. If you're trying to get realtime data for some agency, this is a good place to start.

-------

Some other things we could add:

- Explain what OneBusAway is
- Mention some of the other common transit data formats like SIRI

### PR DESCRIPTION
**Pull request by luqmaan**
I don't know why I didn't just open a PR. 😞 

Closes #58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +32,15 @@ Have something to add or change? Open a [pull request](https://github.com/luqmaa
   - [GTFS Realtime Utilities](#gtfs-realtime-utilities)
 - [SIRI](#siri)
 
+### Getting started
+
+If this is your first time dealing with transit data, you might find these links useful:
+
+- [GTFS](https://developers.google.com/transit/gtfs/) - GTFS is a of text files that contains infrequently changing transit data, like stops, routes, trips, and other schedule data. Most transit agencies typically update their GTFS feed every few months.
+- [GTFS Realtime](https://developers.google.com/transit/gtfs-realtime/) - GTFS Realtime consists of three binary files that contain realtime vehicle positions, realtime arrival information, and service alerts. Transit agencies typically update these files every minute.
+- [Transitland ](https://transit.land/) - A community curated list of GTFS/GTFS-realtime data feeds from around the world. If you're trying to get realtime data for some agency, this is a good place to start.
```
Comment (barbeau) [inline README.md]: I don't think Transitland had any realtime feeds yet, although it's on their roadmap. Transitfeeds.com is the best directory for realtime feeds that I know of.

**Review by barbeau — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (luqmaan) [reply README.md]: Ah that's much better. Swapped them.

**Comment by luqmaan** [comment]
I don't think we should list both transitland and transitfeeds in the Getting Started section.

They're both listed later in the page.

To me, the Getting Started section should be as short as possible and should be the things you should look at if it's your first time hearing about GTFS.

I don't think beginners need to know about both transitland and transitfeeds. They're so similar. And transitfeeds has more data than transitland.

**Comment by barbeau** [comment]
Ok, works for me. Thanks!

**Pull request merged**